osu.Game/Overlays/Options/SliderOption.cs

```diff
@@ -104,13 +105,14 @@ public OsuSliderBar()
                         AutoSizeAxes = Axes.None,
                         RelativeSizeAxes = Axes.None,
                         Masking = true,
-                        BorderColour = new Color4(255, 102, 170, 255),
+                        BorderColour = OsuColor.SliderbarNub,
                         BorderThickness = 3,
                         Children = new[]
                         {
                             new Box
                             {
-                                Colour = new Color4(255, 102, 170, 0),
+                                Colour = new Color4(OsuColor.SliderbarNub.R,

```

Comment:
agreed, and probably a Color4.GreyFrom(0.5f) function to cover grey creation too.